imu_sensor_broadcaster/src/imu_sensor_broadcaster.cpp

```diff
@@ -43,6 +42,10 @@ controller_interface::CallbackReturn IMUSensorBroadcaster::on_init()
 controller_interface::CallbackReturn IMUSensorBroadcaster::on_configure(
   const rclcpp_lifecycle::State & /*previous_state*/)
 {
+  if (!param_listener_->are_params_initialized())
```

Comment:
do we need this in the client code, or should we put this in the library?

Reply:
I think it won't be needed for sure if you call the `initialize_parameters`  on the first instance. If the `get_params` is called, then it throws an exception anyway. 

We can remove the check if you prefer